website/content/specification/values/exploited-weakness.md

```diff
@@ -10,10 +10,11 @@ The weakness causing the Vulnerability. When choosing a value, the most applicab
 
 The Common Weakness Enumeration (CWE) [CWE](https://cwe.mitre.org/about/index.html) which provides descriptions and names for various types of vulnerability weaknesses.
 
-### Example Values:
-- CWE-79
-- CWE-399
+### Examples
+
+- `CWE-79`
+- `CWE-399`
```

Comment:
This breaks pattern with the rest of the valid value lists. Is there a philosophical purpose to display the examples differently than other established valid values?